packages/esbuild-transpiler/vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineProject } from 'vitest/config'
+
+export default defineProject({
+  test: {
+    globals: true,
+    include: ['src/**/*.test.ts'],
+  },
+})
```

Comment:
This project was already included, adding `vitest.config.ts` just ensures it is still included with these changes